apps/wfo-ui-surf/components/WfoServiceTicketsList/WfoServiceTicketsList.tsx

```diff
@@ -0,0 +1,253 @@
+                />
+            ),
+        },
+        jira_ticket_id: {
+            field: SERVICE_TICKET_FIELD_JIRA_ID,
+            name: t('jiraTicketId'),
+            width: '100',
+            //Todo: Fix styling and add color according to State
+            render: (value) => (
+                <EuiFlexGroup>
+                    <EuiFlexItem>{value}</EuiFlexItem>
+                </EuiFlexGroup>
+            ),
+            sortable: true,
+        },
+        title_nl: {
+            field: SERVICE_TICKET_FIELD_TITLE,
+            name: t('titleNl'),
+            width: '300',
+        },
+        process_state: {
+            field: SERVICE_TICKET_FIELD_PROCESS_STATE,
+            name: t('processState'),
+            width: '120',
+            render: (value, object) => (
+                <WfoServiceTicketStatusBadge
+                    serviceTicketState={object.process_state}
+                />
+            ),
+        },
+        opened_by: {
+            field: SERVICE_TICKET_FIELD_OPENED_BY,
+            name: t('openedBy'),
+            width: '180',
+        },
+        start_date: {
+            field: SERVICE_TICKET_FIELD_START_DATE,
+            name: t('startDate'),
+            width: '150',
+            render: (date) => <WfoDateTime dateOrIsoString={date} />,
+            renderDetails: parseIsoString(parseDateToLocaleDateTimeString),
+            clipboardText: parseIsoString(parseDateToLocaleDateTimeString),
+        },
+        create_date: {
+            field: SERVICE_TICKET_FIELD_CREATE_DATE,
+            name: t('createDate'),
+            width: '150',
+            render: (date: string) => (
+                <span>{new Date(date).toLocaleDateString()}</span>
+            ),
+        },
+        last_update_time: {
+            field: SERVICE_TICKET_FIELD_LAST_UPDATE,
+            name: t('lastUpdateTime'),
+            width: '150',
+            render: (date: string) => (
+                <span>{new Date(date).toLocaleString()}</span>
+            ),
+        },
+    };
+
+    const onCriteriaChange = ({
+        page,
+        sort,
+    }: Criteria<ServiceTicketDefinition>) => {
+        if (page) {
+            const { index: pageIndex, size: pageSize } = page;
+            setPageIndex(pageIndex);
+            setPageSize(pageSize);
+        }
+        if (sort) {
+            const { field: sortField, direction: sortDirection } = sort;
+            const sortOrder = getSortDirectionFromString(
+                sortDirection,
+            ) as SortOrder;
+            setDataSorting({
+                field: sortField,
+                sortOrder: sortOrder,
+            });
+        }
+    };
+
+    // Manually handle sorting and pagination of data
+    const findServiceTickets = (
+        serviceTickets: ServiceTicketDefinition[],
+        pageIndex: number,
+        pageSize: number,
+        sortField: keyof ServiceTicketDefinition,
+        sortDirection: 'asc' | 'desc',
```

Comment:
in the types file of the orchestrator-ui-components we have a type that matches this one:

```export enum SortOrder {
    ASC = 'ASC',
    DESC = 'DESC',
}```